docs/time_series_performance_benchmarks.ipynb

```diff
@@ -0,0 +1,156 @@
+{
```

Comment:
<p>I think this sentence isn't complete, should it say "so reducing the size of the AOI makes little impact on run time"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/titiler-cmr/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>